presto-hive-hadoop2/bin/run_hive_alluxio_tests.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+set -euo pipefail -x
+
+. "${BASH_SOURCE%/*}/common.sh"
+
+source presto-product-tests/conf/product-tests-config-hdp3.sh
+
+ALLUXIO_DOCKER_COMPOSE_LOCATION="${INTEGRATION_TESTS_ROOT}/conf/alluxio-docker.yml"
+
+function start_alluxio_containers() {
+  # stop already running containers
+  docker-compose -f "${ALLUXIO_DOCKER_COMPOSE_LOCATION}" down || true
+
+  # catch terminate signals
+  trap termination_handler INT TERM
+
+  # pull docker images
+  if [[ "${CONTINUOUS_INTEGRATION:-false}" == 'true' ]]; then
+    docker-compose -f "${ALLUXIO_DOCKER_COMPOSE_LOCATION}" pull --quiet
+  fi
+
+  # start containers
+  docker-compose -f "${ALLUXIO_DOCKER_COMPOSE_LOCATION}" up -d
+
+  # start docker logs for hadoop container
+  docker-compose -f "${ALLUXIO_DOCKER_COMPOSE_LOCATION}" logs --no-color alluxio-master &
+
+  retry check_alluxio
+}
+
+function check_alluxio() {
+  run_in_alluxio alluxio fsadmin report
+}
+
+function run_in_alluxio() {
+    docker exec -e ALLUXIO_JAVA_OPTS=" -Dalluxio.master.hostname=localhost" \
+    "$(alluxio_master_container)" $@
+}
+
+# Arguments:
+#   $1: container name
+function get_alluxio_container() {
+  docker-compose -f "${ALLUXIO_DOCKER_COMPOSE_LOCATION}" ps -q "$1" | grep .
+}
+
+function alluxio_master_container() {
+  get_alluxio_container alluxio-master
+}
+
+function main () {
+  cleanup_docker_containers
+  start_docker_containers
+
+  export HADOOP_MASTER_IP=$(hadoop_master_ip)
+  export ALLUXIO_BASE_IMAGE="alluxio/alluxio"
+  export ALLUXIO_IMAGE_TAG="2.1.1"
+
+  start_alluxio_containers
```

Comment:
You should start environment with one command.
You still can have two compose yml files and combine them this way:
https://github.com/prestosql/presto/blob/d6cd19a1cba1a5262648af0f6f9232f0f0dde07e/presto-product-tests/conf/docker/singlenode/compose.sh#L7-L11